docs/features/role-based-access-control/brief.md

```diff
@@ -0,0 +1,40 @@
+# Role base access control - Brief
+
+This document aims at providing an overview of Role-based Access Control (RBAC from here on) for the Marxan platform. 
+
+In broad strokes, RBAC assumes the existence of a set of roles, which are associated with different actions that they 
+can perform in the platform. Users can then be associated with roles for specific resources, which will grant them the 
+role privileges for that specific resource.
+
+The current implementation is based on the requirements described in the [User requirements document](https://docs.google.com/document/d/16x4wQYTKov0JkO1ue6iQzJ3-RXpdgQ38ygGJWW86Zo0/edit#heading=h.58xbt2es73r2).
+
+## Roles
+
+In the context of the Marxan platform, we will consider the following roles:
+
+* **Admin**: a generic platform administrator, who can perform **any action for any resource** in the platform.
+* **Owner**: represents a user that can perform any management actions for a specific resource of the platform (e.g. 
+a project). This role is typically attributed to the user who created the resource (i.e. they own the resource).
+* **Contributor**: represents a user with read-write access to a specific resource. Users with this role cannot manage 
+the roles of users associated with a resource or delete the resource.
+* **Viewer**: represents a user with read-only access to a given resource or particular part of the resource (think of `solutions` for scenario, or map tiles)
+
+Note: initially, we thought about considering "Temporal users" as a role, but since this feature will be achieved by 
+using dedicated instances for training, this role will not be implemented.
+
+## Resources
+
+We will consider the following entities as resources subject to RBAC:
+
+1. Organizations
+2. Projects
+3. Scenarios
+
+The entities stated above will work in a hierarchical way: roles granted to users in organizations also apply to all 
+projects owned by said organization, as well as all scenarios in all projects owned by the organization.
+
+However, this hierarchical behavior can be overwritten in a case-by-case basis if needed - e.g. a given user can be 
```

Comment:
is it worth specifying if there is an expectation (which I don't think we do have nor should have) that roles can be "narrowed", for example, granting some user a *contributor* role on an organization but restricting their role to *viewer* on a subset of the projects of the organization, while leaving the default role cascade as default (so the user would be *contributor* by default on any existing and new project within the organization, except those where their role has been reduced to *viewer*).

I would not recommend even planning this at this stage (there are several implications, such as for example being able to fully revoke any role on a specific child resource, or how to deal with cascades in many-to-many relationships between entities), but either way, I recommend to explicitly exclude this if we want to do so.

Reply:
As discussed during today's daily meeting, I'll remove the "hierarchical" bits from this implementation,